src/main/java/mate/academy/bookstore/entity/Book.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.bookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "books")
+public class Book {

```

Comment:
this is not critical for now, but `@Data` is not recommended for use with `@Entity`, details on that:
https://medium.com/@dev.fethi/data-anatation-and-hibernate-n-1-problems-466df982288e